apps/dsp-app/src/app/workspace/resource/values/boolean-value/boolean-value.component.ts

```diff
@@ -4,7 +4,7 @@ import {
     Input,
     OnChanges,
     OnDestroy,
-    OnInit,
+    OnInit
```

Comment:
The idea of the comma is: In case you import a new class you only change one line

Reply:
Yup, that's no secret. It is a question of style and preference: Some hate redundant characters - some like to set it in advance, so they don't have to set it if they add a new class, but set it there again in advance  ;-)